lib/resources/df.rb

```diff
@@ -0,0 +1,28 @@
+# encoding: utf-8
+# author: Jonathan Morley
+
+module Inspec::Resources
+  class DfResource < Inspec.resource(1)
+    name 'df'
+    desc 'Use the df InSpec resource to test file system disk space usage'
+    example "
+      describe df('/') do
+        its('size') { should be >= 32000 }
+      end
+    "
+    attr_reader :partition
+
+    def initialize(partition)
+      @partition = partition
+      @cmd = inspec.command("df #{@partition} --output=size | sed \"/blocks/d; s/ *//g\"")
```

Comment:
I don't think there's any value in storing the command in an instance variable since it's only ever used in the `size` method... other than to perhaps prevent the command from being executed twice.

Instead, we can memoize the size value in the `size` method to avoid this. Example below.